packages/cmr-client/CMRSearchConceptQueue.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const searchConcept = require('./searchConcept');
+
+/**
+ * A class to efficiently list all of the concepts (collections/granules) from
+ * CMR search, without loading them all into memory at once.  Handles paging.
+ *
+ * @typicalname cmrSearchConceptQueue
+ *
+ * @example
+ * const { CMRSearchConceptQueue } = require('@cumulus/cmr-client');
+ *
+ * const cmrSearchConceptQueue = new CMRSearchConceptQueue(
+ *   'my-provider',
+ *   'my-clientId',
+ *   'granule',
+ *   {},
+ *   'json'
+ * );
```

Comment:
Again, I think the cmr-client version of CMRSearchConceptQueue takes a param object, not a list of params.

Reply:
Same as above, this does actually take a param object but the example doesn't match the implementation.